NAMESPACE

```diff
@@ -50,7 +50,6 @@ export(importQIIME2)
 export(importQZA)
 export(inner_join)
 export(left_join)
-export(loadFromBiom)
```

Comment:
Always check. Now loadFromBiom is removed. That is because you forgot to add

```
#' @rdname deprecate
#' @export

```
above loadFromBiom in deprecate.R